packages/twitch/src/API/Helix/User/HelixUser.ts

```diff
@@ -124,6 +125,13 @@ export class HelixUser implements UserIdResolvableType, UserNameResolveableType
 		return this._data.view_count;
 	}
 
+	/**
+	 * The date when the user was created, i.e. when they registered on Twitch.
+	 */
+	get creationDate(): string {
+		return this._data.created_at;
```

Comment:
All timestamps in this library are wrapped in a `Date` object. Please change this getter accordingly.